### PR TITLE
site: fixed twitter image

### PR DESCRIPTION
**Pull request by jkowalski**

### CODE REVIEW

**Comment by ntolia** [comment]
A few concerns. This is what kopia.io already has:
```
<meta property="og:description" content="Fast And Secure Open Source Backup" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/" />
<meta property="og:site_name" content="Kopia" />
<meta itemprop="name" content="Kopia">
<meta itemprop="description" content="Fast And Secure Open Source Backup">

<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:image" content="/featured-background.jpg"/>
<meta name="twitter:title" content="Kopia"/>
<meta name="twitter:description" content="Fast And Secure Open Source Backup"/>```

How does this change conflict? Are we getting duplicate tags?

File: site/themes/docsy/layouts/partials/head.html [hunk 1]
```diff
@@ -14,7 +14,6 @@
 {{- template "_internal/opengraph.html" . -}}
 {{- template "_internal/google_news.html" . -}}
 {{- template "_internal/schema.html" . -}}
-{{- template "_internal/twitter_cards.html" . -}}
```
Comment (ntolia) [inline site/themes/docsy/layouts/partials/head.html]: Ah nevermind, does this take care of the default tags?

Comment (jkowalski) [reply site/themes/docsy/layouts/partials/head.html]: yes

Comment (jkowalski) [reply site/themes/docsy/layouts/partials/head.html]: btw, PTAL @#335 when it is deployed

**Pull request closed without merging**